packages/base/src/features/F6Navigation.js

```diff
@@ -54,24 +54,20 @@ class F6Navigation {
 			const nextIndex = this.groups.indexOf(this.selectedGroup);
 			let nextElement = null;
 
-			if (nextIndex > -1) {
-				if (nextIndex - 1 < 0) {
-					nextElement = this.groups[this.groups.length - 1];
```

Comment:
0 case was the same as the outer `if`s else branch

Reply:
You're right.